.github/workflows/enos-run.yml

```diff
@@ -274,7 +274,14 @@ jobs:
           path: ${{ env.ENOS_DEBUG_DATA_ROOT_DIR }}
           retention-days: 30
       - name: Destroy Enos scenario
-        if: ${{ always() && steps.run_retry.outcome == 'failure' }}
+        id: destroy
+        continue-on-error: true
+        if: ${{ always() }}
+        run: |
+          export ENOS_VAR_enos_user=$GITHUB_ACTOR && \
+          enos scenario destroy --timeout 60m0s --chdir ./enos ${{ matrix.filter }}
```

Comment:
Just noticed the 60m timeout that we've had. Is it necessary to have such a big timeout here?

Reply:
It might not be, but I think it's fine for now. I don't believe we've ever hit that timeout.